include/nlohmann/detail/output/serializer.hpp

```diff
@@ -811,6 +811,18 @@ class serializer
         return state;
     }
 
+    number_unsigned_t remove_sign(number_unsigned_t x)
+    {
+        assert(false); // LCOV_EXCL_LINE
+        return x; // LCOV_EXCL_LINE
+    }
+
+    inline number_unsigned_t remove_sign(number_integer_t x) noexcept
+    {
+        assert(x < 0 && x < (std::numeric_limits<number_integer_t>::max)());
```

Comment:
This library uses alternative operator keywords. Please change to:

```suggestion
        assert(x < 0 and x < (std::numeric_limits<number_integer_t>::max)());
```

Reply:
Done